test/test_batch.cpp

```diff
@@ -0,0 +1,134 @@
+/***************************************************************************
+* Copyright (c) Johan Mabille, Sylvain Corlay, Wolf Vollprecht and         *
+* Martin Renou                                                             *
+* Copyright (c) QuantStack                                                 *
+*                                                                          *
+* Distributed under the terms of the BSD 3-Clause License.                 *
+*                                                                          *
+* The full license is in the file LICENSE, distributed with this software. *
+****************************************************************************/
+
+#include <functional>
+
+#include "test_utils.hpp"
+
+template <class B>
+class batch : public testing::Test
```

Comment:
FTR, I would probably name it `batch_test` or something similar, calling it `batch` invites confusion with `xsimd::batch` and may produce head-scratching.

Reply:
That was its initial name. However, the complete output is something like:

```
[----------] 9 tests from sse/batch/double, where TypeParam = xsimd::batch<double, 2ul>
[ RUN      ] sse/batch/double.load_store

```

I found the `batch_test` name a bit repetitive in the output. But I can definitely revert that